src/busctl/busctl.c

```diff
@@ -1978,46 +2084,26 @@ static int parse_argv(int argc, char *argv[]) {
         return 1;
 }
 
-static int busctl_main(sd_bus *bus, int argc, char *argv[]) {
-        assert(bus);
-
-        if (optind >= argc ||
-            streq(argv[optind], "list"))
-                return list_bus_names(bus, argv + optind);
-
-        if (streq(argv[optind], "monitor"))
-                return monitor(bus, argv + optind, message_dump);
-
-        if (streq(argv[optind], "capture"))
-                return capture(bus, argv + optind);
-
-        if (streq(argv[optind], "status"))
-                return status(bus, argv + optind);
-
-        if (streq(argv[optind], "tree"))
-                return tree(bus, argv + optind);
-
-        if (streq(argv[optind], "introspect"))
-                return introspect(bus, argv + optind);
-
-        if (streq(argv[optind], "call"))
-                return call(bus, argv + optind);
-
-        if (streq(argv[optind], "get-property"))
-                return get_property(bus, argv + optind);
-
-        if (streq(argv[optind], "set-property"))
-                return set_property(bus, argv + optind);
-
-        if (streq(argv[optind], "help"))
-                return help();
+static int busctl_main(int argc, char *argv[]) {
+
+        static const Verb verbs[] = {
+                { "call",         4,        VERB_ANY, 0, call           },
+                { "capture",      VERB_ANY, VERB_ANY, 0, verb_capture   },
+                { "help",         VERB_ANY, VERB_ANY, 0, verb_help      },
+                { "introspect",   2,        2,        0, introspect     },
```

Comment:
`busctl introspect org.freedesktop.systemd1 /org/freedesktop/systemd1/unit/unbound_2danchor_2etimer` should work, but it doesn't after this change.